app/README/ta_databricks_account.conf.spec

```diff
@@ -0,0 +1,8 @@
+[<name>]
+databricks_instance = 
+client_id = 
+tenant_id = 
+client_secret = 
+access_token =
+cluster_name = 
+databricks_access_token = 
```

Comment:
can we name it as `token`?... our upcoming Python SDK will use the same authentication semantics as described in this document: https://github.com/databricks/databricks-sdk-go#authentication, so I highly advise to change the names of configuration properties to have a Unified Authentication for all databricks clients.

also it's not clear if `access_token` and `databricks_access_token` are the same.

Reply:
@nfx - As suggested by @alexott, we will add prefix "pat" and "aad" for both authentication types. So new variables would be:

- client_id --> aad_client_id
- tenant_id --> aad_tenant_id
- client_secret --> aad_ client_secret
- access_token --> aad_ access_token
- databricks_access_token --> pat_access_token

Let me know if you have any suggestions here.

Reply:
what is `access_token`? is it PAT?